libs/blocks/tabs/tabs.js

```diff
@@ -109,64 +136,57 @@ const init = (block) => {
   if (tabListItems) {
     const btnClass = [...block.classList].includes('quiet') ? 'heading-xs' : 'heading-xs';
     tabListItems.forEach((item, i) => {
-      const tabName = getStringKeyName(item.textContent);
+      const tabName = config.id ? i+1 : getStringKeyName(item.textContent);
       const tabBtnAttributes = {
         role: 'tab',
         class: btnClass,
-        id: `tab-${initCount}-${tabName}`,
+        id: `tab-${tabsId}-${tabName}`,
         tabindex: '0',
         'aria-selected': (i === 0) ? 'true' : 'false',
-        'aria-controls': `tab-panel-${initCount}-${tabName}`,
+        'aria-controls': `tab-panel-${tabsId}-${tabName}`,
       };
       const tabBtn = createTag('button', tabBtnAttributes);
       tabBtn.innerText = item.textContent;
       tabListContainer.append(tabBtn);
 
       const tabContentAttributes = {
-        id: `tab-panel-${initCount}-${tabName}`,
+        id: `tab-panel-${tabsId}-${tabName}`,
         role: 'tabpanel',
         class: 'tabpanel',
         tabindex: '0',
-        'aria-labelledby': `tab-${initCount}-${tabName}`,
+        'aria-labelledby': `tab-${tabsId}-${tabName}`,
       };
       const tabListContent = createTag('div', tabContentAttributes);
-      tabListContent.setAttribute('aria-labelledby', `tab-${initCount}-${tabName}`);
+      tabListContent.setAttribute('aria-labelledby', `tab-${tabsId}-${tabName}`);
       if (i > 0) tabListContent.setAttribute('hidden', '');
-
       tabContentContainer.append(tabListContent);
     });
     tabListItems[0].parentElement.remove();
   }
 
-  // Tab Config
-  const config = { 'tab-id': initCount };
-  const configRows = [].slice.call(rows);
-  configRows.splice(0, 1);
-  if (configRows) {
-    configRows.forEach((row) => {
-      const rowKey = getStringKeyName(row.children[0].textContent);
-      const rowVal = row.children[1].textContent.trim();
-      config[rowKey] = rowVal;
-      row.remove();
-    });
-  }
-
   // Tab Sections
   const allSections = Array.from(rootElem.querySelectorAll('div.section'));
-  allSections.forEach((e) => {
+  allSections.forEach((e, i) => {
     const sectionMetadata = e.querySelector(':scope > .section-metadata');
     if (!sectionMetadata) return;
-    const metadata = sectionMetadata.querySelectorAll(':scope > div');
-    [...metadata].filter((d) => getStringKeyName(d.children[0].textContent) === 'tab')
-      .forEach((d) => {
-        const metaValue = getStringKeyName(d.children[1].textContent);
-        const section = sectionMetadata.closest('.section');
-        const assocTabItem = rootElem.querySelector(`#tab-panel-${initCount}-${CSS.escape(metaValue)}`);
-        if (assocTabItem) assocTabItem.append(section);
-      });
+    const rows = sectionMetadata.querySelectorAll(':scope > div');
+    rows.forEach((row) => {
+      const key = getStringKeyName(row.children[0].textContent);
+      if (key != 'tab') return;
```

Comment:
Is there a reason you aren't using strict inequality?

Reply:
Derp, nope - just an oversight. This is updated, thanks!